src/app/globals.css

```
@@ -86,16 +236,320 @@
+  }
+
+  .flat-control:focus-within,
+  .flat-control:focus {
+    background: #2f333d;
+    border-color: #ffb08a;
+    box-shadow: 0 12px 30px rgba(0, 0, 0, 0.24);
+  }
+
+  .status-pill {
+    @apply inline-flex items-center gap-2 rounded-full border px-3 py-1.5 text-xs font-semibold;
+    background: rgba(47, 51, 61, 0.82);
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)